finite_state_machine/state_machine.py

```diff
@@ -21,6 +40,37 @@ class Transition(NamedTuple):
     on_error: Union[bool, int, str]
 
 
+class TransitionMeta(object):
```

Comment:
Think this might be importing a lot of unnecessary complexity from django-fsm.

Reply:
I agree with you. I stuck with the django-fsm design here.